src/Shared/ExceptionHandling.cs

```diff
@@ -351,6 +353,28 @@ internal static void DumpExceptionToFile(Exception ex)
                 }
             }
         }
+
+        /// <summary>
+        /// Returns the content of any exception dump files modified
+        /// since the provided time, otherwise returns an empty string.
+        /// </summary>
+        internal static string ReadAnyExceptionFromFile(DateTime fromTimeUtc)
+        {
+            string result = "";
```

Comment:
I assume using a StringBuilder here isn't worth much since this is an error-handling about-to-shut-down path and we don't care about GC? Only counterpoint I can think of is "what if we're already under memory pressure and that's why something crashed".

Reply:
I don't think there's a clear winner for a small number of appends if done only once, as StringBuilder resizes without knowledge of what is to come, so it could do several resizes, and those need to be GC'd too. Fair enough though, I changed.